networking/eth2/src/integration-test/java/tech/pegasys/teku/networking/eth2/GossipMessageHandlerIntegrationTest.java

```diff
@@ -341,7 +368,7 @@ private NodeManager createNodeManager(final Consumer<Eth2P2PNetworkBuilder> netw
   }
 
   public static Stream<Arguments> getEncodings() {
-    final List<GossipEncoding> encodings = List.of(GossipEncoding.SSZ, GossipEncoding.SSZ_SNAPPY);
+    final List<GossipEncoding> encodings = List.of(GossipEncoding.SSZ_SNAPPY);
```

Comment:
Probably shouldn't make this change as part of this PR.

Reply:
Reversed.